builder/chroots.go

```diff
@@ -256,7 +256,7 @@ src=%s
 		return errors.Wrap(err, "couldn't fix os-release file")
 	}
 
-	fmt.Println("Creating chroots for bundles")
+	fmt.Printf("Creating chroots for %d bundles\n", len(set))
```

Comment:
Since os-core is already done, maybe you want len(set)-1.

Reply:
Just throw this print down below near the for loop that feed the channel, so this will be clearer.